docs/docker.md

```diff
@@ -1,80 +1,38 @@
 # Run Percona XtraBackup 8.0 in a Docker container
 
-Docker allows you to run applications in a lightweight unit called a
-container.
+Docker containers are built from Docker images, which are snapshots of the configuration needed to run an applications, such as the code, and libraries. With Docker, you can build, deploy, run, update, and manage containers, which isolate applications from the host system. A Docker container lets you work with Percona XtraBackup without installing the product on a local drive. You may run *Percona Server for MySQL* in one container and *Percona XtraBackup* in another.
 
-You can run *Percona XtraBackup* in a Docker container without installing
-the product. All required libraries are available in
-the container. Being a lightweight execution environment, Docker containers
-enable creating configurations where each program runs in a separate container. You may run
-*Percona Server for MySQL* in one container and *Percona XtraBackup* in
-another. Docker images offer a range of options.
+For this document, container refers to the Docker container and instance refers to the database server in the container.
 
-Create a Docker container based on a Docker image. Docker images for
-Percona XtraBackup are hosted publicly on [Docker Hub](https://hub.docker.com/r/percona/percona-xtrabackup).
+*Percona XtraBackup* has an official Docker image hosted on [Docker Hub](https://hub.docker.com/r/percona/percona-xtrabackup/). If you want the latest version, use the `latest` tag. You can reference a specific version using the [Docker tag filter for the 8.0 versions](https://registry.hub.docker.com/r/percona/percona-xtrabackup/tags?page=1&name=8.0).
```

Comment:
I thought we weren't doing
*Percona XtraBackup*

Reply:
fixed